driver/src/util.rs

```diff
@@ -30,7 +30,9 @@ pub fn hash_consistency_check(
         return Err(DriverError::new(
             &format!(
                 "Pending {} hash from contract ({}), didn't match the one found in db ({})",
-                flux_type, hash_from_contract, hash_calculated
+                flux_type, 
+                serde_json::to_string(&hash_from_contract).unwrap(),
```

Comment:
You should be able to use `{:#x}` in the format string directly, since H256 implements LowerHex (https://doc.rust-lang.org/nightly/core/fmt/trait.LowerHex.html)